src/js/entities-service/entities/program-actions.js

```diff
@@ -129,15 +129,25 @@ const Collection = BaseCollection.extend({
       data: JSON.stringify({ data }),
     });
   },
+  isVisibleToCurrentUser(action) {
+    const visibleToList = action.get('_visible_to');
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    const currentUserTeamId = currentUser.get('_team');
```

Comment:
I think we should us `getTeam()` here.  we can use `.id`

Reply:
Updated 👍